bun-tests/fake-snippets-api/routes/package_releases/update.test.ts

```diff
@@ -92,18 +92,20 @@ test("update package release - handle is_latest flag", async () => {
   })
 
   // Update second release to be latest
-  await axios.post("/api/package_releases/update", {
-    package_release_id: release2.data.package_release.package_release_id,
-    is_latest: true,
-  })
-
-  // Verify first release is no longer latest
-  const firstRelease = db.packageReleases.find(
-    (pr) =>
-      pr.package_release_id ===
-      release1.data.package_release.package_release_id,
-  )
-  expect(firstRelease?.is_latest).toBe(false)
+  await axios
+    .post("/api/package_releases/update", {
+      package_release_id: release2.data.package_release.package_release_id,
+      is_latest: true,
```

Comment:
No change needed, but want to explain the game plan:

The plan in the future is to make it so that the user has to set `is_locked: true` before a package version can be tagged latest. This way, saving a version that wasn't explicitly released will never break people's packages.

We will also support installing `package@unstable`/`package@prerelease`,`package@unstable-latest`, these tags will install the most recent version that is not yet locked